app/javascript/components/NamedAvatar.js

```diff
@@ -16,17 +19,78 @@ const styles = {
   name: {
     fontWeight: 600,
   },
-  subtitle: {},
+  remove: {
+    marginLeft: '20px',
+    cursor: 'pointer',
+  },
+  dialogActionsLink: {
+    padding: '15px',
+    cursor: 'pointer',
+    fontSize: '16px',
+  },
 }
 
-const NamedAvatar = props => (
+const dialogActions = (togglePopover, onRemove) => [
+  <a
+    style={styles.dialogActionsLink}
+    onClick={() => {
+      togglePopover('removeUserFromEvent', false)
```

Comment:
[We're being truthy](https://github.com/zendesk/volunteer_portal/blob/b87c36e787789117f16e36220aa5186a7a50430d/app/javascript/actions/index.js#L29) in the `togglePopover` function definition with the second argument (`data`).

But, i think the second argument should either be provided with `null` or excluded entirely since it's only meant to save retain popover specific data in the state. I should document that 😓. 

So `togglePopover('removeUserFromEvent')` is fine